hpy/devel/include/universal/hpy.h

```diff
@@ -13,7 +13,8 @@ typedef struct _HPy_s HPy;
 typedef struct _HPyContext_s *HPyContext;
 struct _object;  /* that's PyObject inside CPython */
 typedef struct _object *(*_HPy_CPyCFunction)(struct _object *self,
-                                             struct _object *args);
+                                             struct _object *args,
```

Comment:
This is wrong. `_HPy_CPyCFunction` is a direct translation of CPython's `PyCFunction`, so it should stay the same

Reply:
Fixed!